go.sum

```diff
@@ -1,3 +1,5 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898 h1:SC+c6A1qTFstO9qmB86mPV2IpYme/2ZoEQ0hrP+wo+Q=
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
It has been argued that bundling Helm will make our dependencies difficult to manage.
And glancing at these changes, it does seem like there are *a lot* of new dependencies here.

But many of these changes in `go.sum` seem to be updates of version of libraries that we already depend on.

Reply:
`go mod tidy` reduces the number of records that are in `go.sum`.
Unfortunately, after running `./hack/update-all.sh` there are added again.

Reply:
That's a shame, I wonder if its because bazel is using a differnt version of Go than you?